scripts/trivia.coffee

```diff
@@ -2,31 +2,56 @@
 #   start trivia round 
 #
 # Commands:
-#   !trivia
 #   trivia help
 
+points = {}
+
 module.exports = (robot) ->
 
+  robot.brain.on 'loaded', ->
+
+    #points are stored as public object in redis
+	#to store private, use robot.brain.set/get
+    robot.brain.data.points ||= {}
+    points = robot.brain.data.points
+
+  #default values
   triviaOn = false
   hintInterval = null
+  currentSolution = ''
+
   #url for questions and answers
   url = "http://jservice.io/api/random"
-  currentSolution = ''
 
+
+  #----------------Current Question----------------#
   triviaStart = (msg) ->
     triviaOn = true
     msg.send "Searching for a question..."
```

Comment:
You should update this so that when the trivia starts it tells you how to answer and what not. Its really hard to know how to answer without looking trough the source code.